src/frontends/onnx/frontend/src/op/layer_normalization.cpp

```diff
@@ -43,58 +32,42 @@ ov::OutputVector layer_normalization(const ov::frontend::onnx::Node& node) {
                      num_inputs == 2 || num_inputs == 3,
                      "LayerNormalization expects 2 or 3 input tensors. Got: ",
                      num_inputs);
+    CHECK_VALID_NODE(node,
+                     node.get_outputs_size() == 1,
+                     "LayerNormalization expects 1 output tensor to be used in a model, other configurations are used "
+                     "for training and are not supported. Got: ",
+                     node.get_outputs_size(),
+                     " outputs.");
 
-    const auto& X = inputs.at(0);
-    const auto& Scale = inputs.at(1);
-
-    auto axis = node.get_attribute_value<std::int64_t>("axis", -1);
-    double epsilon = node.get_attribute_value<double>("epsilon", 1e-5);
-    int64_t stash_type_i =
-        node.get_attribute_value<int64_t>("stash_type",
-                                          static_cast<int64_t>(TensorProto_DataType::TensorProto_DataType_FLOAT));
+    auto default_stash_type_i = static_cast<int64_t>(TensorProto_DataType::TensorProto_DataType_FLOAT);
```

Comment:
Conversion code shouldn't rely on ONNX-specific data structures that should be isolated under the decoder. Is the TensorProto_DataType::TensorProto_DataType_FLOAT usage unavoidable?

Reply:
This is default value of `stash_type` attribute. It has to do with operation semantics only.